app/models/device.rb

```diff
@@ -0,0 +1,76 @@
+class Device < ApplicationRecord
+  belongs_to :user
+
+  validates :token, uniqueness: { scope: %i[user_id platform app_bundle] }
+
+  def create_notification(title, body, payload)
+    case platform
+    when "iOS"
+      ios_notification(title, body, payload)
+    when "Android"
+      android_notification(title, body, payload)
+    end
+  end
+
+  private
+
+  def ios_notification(title, body, payload)
+    n = Rpush::Apns2::Notification.new
+    # [@forem/oss] `.where().first` is necessary because we use Redis data storage
```

Comment:
We don't really have a `@forem/oss` team anymore, at least not in the same sense as before. I'd probably tag this `@forem/backend`.

Reply:
What is this tagging notation: does Github alert users based on code comments?!?

Reply:
Nope, but we used it to remember who originally added a comment/which team feels responsible for addressing a TODO.

Reply:
✅